CMakeLists.txt

```diff
@@ -43,281 +44,260 @@ else()
     set(PROJECT_DLLVERSION)
     set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_INSTALL_PREFIX}/lib/cmake/ /usr/lib/cmake)
 endif()
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}")
 
 option(USE_SSE    "Build tiny-cnn with SSE library support" ON)
 option(USE_AVX    "Build tiny-cnn with AVX library support" ON)
+option(USE_AVX2   "Build tiny-cnn with AVX2 library support" OFF)
 option(USE_TBB    "Build tiny-cnn with TBB library support" OFF)
 option(USE_OMP    "Build tiny-cnn with OMP library support" OFF)
 option(USE_NNPACK "Build tiny-cnn with NNPACK library support" OFF)
+option(USE_OPENCV "Build tiny-cnn with OpenCV library support" OFF)
```

Comment:
why make it an option as it is automatically set in the BUILD_EXAMPLES section ?